src/plugins/data/public/query/timefilter/get_time.ts

```diff
@@ -18,10 +18,10 @@
  */
 
 import dateMath from '@elastic/datemath';
-import { TimeRange } from '../../../common';
+import { IIndexPattern } from 'src/plugins/data/public';
```

Comment:
The import is from within the plugin
I think you could do `import { IIndexPattern } from '../..';`

Reply:
Thanks my auto import pulls from the aliases by default.

Reply:
59aa8a6